src/ServiceManagement/Compute/Commands.ServiceManagement/Generated/Deployment/InvokeAzureDeploymentBeginCreatingMethod.cs

```diff
@@ -0,0 +1,49 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System.Management.Automation;
+using Microsoft.WindowsAzure.Management.Compute;
+
+namespace Microsoft.WindowsAzure.Commands.Compute.Automation
+{
+    [Cmdlet("Invoke", "AzureDeploymentBeginCreatingMethod")]
+    public class InvokeAzureDeploymentBeginCreatingMethod : ComputeAutomationBaseCmdlet
+    {
+        [Parameter(Mandatory = true)]
+        public System.String ServiceName { get; set; }
+
+        [Parameter(Mandatory = true)]
+        public Microsoft.WindowsAzure.Management.Compute.Models.DeploymentSlot DeploymentSlot { get; set; }
```

Comment:
I don't think we want to surface the types from the management client, but it would be simple to generate PowerShell models for these and either generate converters or wrap them in the generated code.  Alternately, if we always use GetValueFromPipelineByPropertyName and ensure that property and parameter names match up, the need for wrapping the types would be minimized